pkg/pipeline/transform/transform_network.go

```diff
@@ -123,6 +123,24 @@ func (n *Network) Transform(inputEntry config.GenericMap) (config.GenericMap, bo
 					outputEntry[rule.Output+"_HostName"] = kubeInfo.HostName
 				}
 			}
+		case api.OpAddKubernetesInfra:
+			srcKubeInfo, err := kubernetes.Data.GetInfo(fmt.Sprintf("%s", outputEntry["SrcAddr"]))
+			if err != nil {
+				logrus.WithError(err).Tracef("can't find kubernetes info for IP %v", outputEntry["SrcAddr"])
+				continue
+			}
+			dstKubeInfo, err := kubernetes.Data.GetInfo(fmt.Sprintf("%s", outputEntry["DstAddr"]))
+			if err != nil {
+				logrus.WithError(err).Tracef("can't find kubernetes info for IP %v", outputEntry["DstAddr"])
```

Comment:
Is it OK that `SrcAddr` and `DstAddr` are hard-coded? Should these field names be configurable?

Reply:
I added dedicated structure to configure rules.
I did not change other rules because I don't want to introduce API breaking change in this PR but I will create a following PR with the changes in the other rules.